src/elysium/tx.h

```diff
@@ -14,6 +15,8 @@ class CTransaction;
 #include "../uint256.h"
 #include "../utilstrencodings.h"
 
+#include <secp256k1/include/Scalar.h>
```

Comment:
The right way to include this one is just `#include <Scalar.h>` due to it is an external library.

Reply:
Removed